### PR TITLE
Plot grid modif

### LINKED ISSUE #1297: ENH: Add keyword add_lines in method plot_grid in gridmapdisplay
On occasion you would like to have the grid lines but not the coast maps in the plot. Separating the two would be useful (see https://github.com/MeteoSwiss/pyart/blob/dev/pyart/graph/gridmapdisplay.py)

### PR DESCRIPTION
**Pull request by jfigui**
Closes #1297.
Added keyword add_lines in plot_grid in gridmapdisplay.py and plot_ppi_map in radarmapdisplay.py.
General linting of gridmapdisplay and radarmapdisplay.

### CODE REVIEW

File: pyart/graph/gridmapdisplay.py [hunk 1]
```diff
@@ -252,6 +248,16 @@ def plot_grid(self, field, level=0, vmin=None, vmax=None,
             ax.add_feature(coastlines, linestyle='-', edgecolor='k',
                            linewidth=2)
 
+        if add_lines:
```
Comment (mgrover1) [inline pyart/graph/gridmapdisplay.py:251]: Could we name this `add_grid_lines` to make it a bit more clear what sort of lines are being added?

File: pyart/graph/radarmapdisplay.py [hunk 2]
```diff
@@ -318,6 +316,12 @@ def plot_ppi_map(
             ax.coastlines(resolution=resolution)
             ax.add_feature(states_provinces, edgecolor='gray')
 
+        if add_lines:
```
Comment (mgrover1) [inline pyart/graph/radarmapdisplay.py:319]: Same here as above

**Review by mgrover1 — CHANGES_REQUESTED** [review]
One small suggestion... other than that, this looks great! Thanks @jfigui

**Review by zssherman — APPROVED** [review]
Looks good to me with the changes, and thanks for the PEP8 changes etc as well!

**Review by mgrover1 — APPROVED** [review]
Good to merge!

**Pull request merged**